specs/BrowserTaskManager.md

```diff
@@ -0,0 +1,69 @@
+# Background
+The WebView2 team has been asked to provide an API to expose the Browser Task
+Manager. At the current time, it is expected that the user will close the Task 
+Manager manually, so we do not need to provide an API to close it.
+
+In this document we describe the updated API. We'd appreciate your feedback.
+
+# Description
+We propose extending `CoreWebView2` to provide a `ShowTaskManager` method.
```

Comment:
Is the task manager window per WebView2 or per browser process? Does it make sense to put it on the environment instead of WebView2?

Reply:
They are per browser process, I mainly put it off the WebView2 as that is where `OpenDevToolsWindow()` is located (and thus where users can access the browser task manager manually via context menu) and we have easy access to the Browser pointer there to call the internal ShowTaskManager method. 

I could expose the API off the environment, it would just have a call to `GetWebView()->OpenTaskManager()` from one of the `created_hosts_`. Do you think it is worth switching it to the environment?

Reply:
Oh sure. Being consistent with OpenDevToolsWindow in name and location seems like a good idea. Should it be OpenTaskManager**Window** to better match OpenDevToolsWindow (and they both go on the ICoreWebView2)?

Reply:
I think that makes sense, renamed to `OpenTaskManagerWindow` now